Source/VMware.vSphereDSC/Tests/Integration/VMHostVss.Integration.Tests.ps1

```diff
@@ -130,7 +128,7 @@ try {
 
             It 'Should return $true when Test-DscConfiguration is run' {
                 # Arrange && Act && Assert
-                Test-DscConfiguration | Should -Be $true
+                { Test-DscConfiguration } | Should -Be $true
```

Comment:
For Test-DscConfiguration the brackets aro not needed as we are not testing if the cmdlet has thrown.

Reply:
Changed